### ISSUE TITLE
Cannot change the admin user and password

### ISSUE THREAD

**Issue by red-smeg** [owncloud-docker/server#76]
installed via docker-compose.
did a docker-compose up -d
logged into the admin user - no problems.

did a docker-compose down
changed the .env username and password values
did a docker-compose up -d build 

but login only accepts the admin user.

Tried docker-compose rm and docker-compose up -d --force-recreate --build

**Comment by tboerger** [comment]
The admin user gets only created initially and won't be changed afterward anymore. This is the expected behavior. If you want to change the admin credentials you have to execute the `occ` command.

**Comment by red-smeg** [comment]
OK but shouldn't container destruction override that ?

**Comment by tboerger** [comment]
If you also delete the named volumes it works. But since you have not removed the named volumes the state have been still persisted.

**Comment by yebo29** [comment]
This also worked for me - delete the docker volumes `docker volume prune`. Of course, this was a brand-new install so didn't care about losing data. If you have data you don't want to lose then backup first!!!

**Comment by Gregliest** [comment]
I found this behavior to be quite surprising. I spent a couple of hours basically doing the steps that @red-smeg outlined above, before finding this thread. I think it's even more cryptic because at least some of the other vars are taken into account between deploys (including the others in the example .env). 

I think the documentation should call this situation out at the very least. I was following these [instructions](https://doc.owncloud.com/server/admin_manual/installation/docker/#logging-in). Are they open source somewhere? I'm happy to submit a pull request.
